src/driver/include/driver/DefaultDriver.hpp

```diff
@@ -43,8 +43,8 @@ class DefaultDriver {
         YamlSource workloadSourceType = YamlSource::kFile;
         std::string workloadSource;  // either file name or yaml
 
-        std::string metricsFormat;
-        std::string metricsOutputFileName;
+        std::vector<std::string> metricsFormats;
+        std::vector<std::string> metricsOutputFileNames;
```

Comment:
I think `std::ordered_set` (or just `std::set`) since we don't want/need dupes

also make sure to `#include <set>` or whatever container type header you need.